rosette/src/Import.cc

```diff
@@ -0,0 +1,122 @@
+#include "Ob.h"
+#include "Code.h"
+#include "Prim.h"
+#include "Number.h"
+#include "Expr.h"
+#include "RblAtom.h"
+#include "Proc.h"
+#include "Method.h"
+#include "Pattern.h"
+#include "Table.h"
+
+#include "CommandLine.h"
+#include "Export.h"
+
+#include <google/protobuf/text_format.h>
+#include <Ob.pb.h>
+
+#include <fstream>
+#include <string>
+#include <map>
+
+
+ObjectCodePB::ObjectCode importObjectCode;
+
+pOb createRosetteObject(const ObjectCodePB::Object & ob) {
+
+    switch(ob.type()) {
```

Comment:
Indentation seems off here.

Reply:
It is common with C++ to not indent the cases in a switch statement. This is what we've chosen to do in Rosette.